### DISCUSSION TITLE
Hi all and thank you so much for all your hard work, i just had a client update and now the game wont load (cant grab cookies) no maint expected at the moment till valentines event.

### DISCUSSION THREAD

**Discussion by Jazzyguy67** [fedorovvl/tso_client#172] [General]
Jazz

**Comment by Nubwos** [comment]
Me and 2 other players who use the client too :(

**Comment by AlbertoMassanet** [comment]
Me too. Outdated client error.

**Comment by Kshyhoo67** [comment]
Uncheck _Upstream SWF_ option, the client will download the update.

**Reply by Jazzyguy67** [reply]
Perfect Kshyhoo67 thank you :)

**Reply by X-Xadro** [reply]
Yes this worked for me, thanks

**Comment by BasilMas** [comment]
Read wiki plz
https://github.com/fedorovvl/tso_client/wiki/Maintenance-and-upstream-SWF

**Comment by 00dcy00** [comment]
今天又可以正常进入了，非常棒。
